src/plugin/cli/build.gradle.kts

```diff
@@ -63,9 +63,8 @@ kotlin {
     sourceSets {
         val commonMain by getting {
             dependencies {
+                implementation(project(":src:compiler:lib"))
                 implementation(project(":src:plugin:arguments"))
-                implementation(project(":src:compiler:core"))
-                implementation(project(":src:converter:openapi"))
                 implementation(CLI_LIB)
             }
         }
```

Comment:
`cli` depends on the `lib` module, using the libraries exposed for all platforms. This alleviates the 'weird' need for `jsMain` only to depend on the `lib` module.